src/pypi_attestations/_impl.py

```diff
@@ -191,6 +191,31 @@ def sign(cls, signer: Signer, dist: Distribution) -> Attestation:
         except ConversionError as e:
             raise AttestationError(str(e))
 
+    @property
+    def claims(self) -> dict[x509.ObjectIdentifier, str]:
```

Comment:
Maybe `claims()` is too generic here. A couple of options:
- `certificate_claims()`
- `claims_in_certificate()`
- `fulcio_claims()`

But I'm not sure which one would be most clear.

Reply:
I prefer `fulcio_claims` because it also hints that the response is going to be tied to Fulcio.

Reply:
Let's go with `certificate_claims` -- IMO that avoids the assumption that the CA will always be Fulcio, since that's not guaranteed (Fulcio is just one possible Sigstore-compatible CA).